src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1053,6 +1053,14 @@ public String get(PrepareItemCraftEvent e) {
 				}
 			}, 0);
 		}
+		// CraftItemEvent
+		EventValues.registerEventValue(CraftItemEvent.class, ItemType.class, new Getter<ItemType, CraftItemEvent>() {
+			@Override
+			@Nullable
+			public ItemType get(CraftItemEvent e) {
```

Comment:
You can just return an ItemStack. We need to be able to remember lore, names, enchantments, etc. Skript will convert an ItemStack to an ItemType if the user attempts to compare an ItemStack with an ItemType.